internal/tiltfile/files.go

```diff
@@ -81,12 +79,6 @@ func (s *tiltfileState) execLocalCmd(t *starlark.Thread, cmd model.Cmd, logOutpu
 		c.Stderr = io.MultiWriter(stderr, logOutput)
 	}
 
-	// if Tilt was invoked with `tilt up --port=XXXXX`, local() calls to use the Tilt API will fail due to trying to
-	// connect to the default port, so explicitly populate the TILT_PORT environment variable if it isn't already
-	addEnvIfNotPresent(c, "TILT_PORT", strconv.Itoa(int(s.webPort)))
-	// some Tilt commands, such as `tilt dump engine`, also require the host
-	addEnvIfNotPresent(c, "TILT_HOST", string(s.webHost))
```

Comment:
FYI the whole `addEnvIfNotPresent` func can be removed since it's no longer used here (line ~306 - can't comment directly on it in GitHub)

Reply:
Oh the linter caught it first 😂

Reply:
fixed!